test/ng/compileSpec.js

```diff
@@ -4193,7 +4194,7 @@ describe('$compile', function() {
       dealoc(element);
       compile('<div><span my-component>');
       expect(componentScope).not.toBe(firstComponentScope);
-      expect(componentScope.$$isolateBindings).toBe(first$$isolateBindings);
+      expect(componentScope.$$isolateBindings).toEqual(first$$isolateBindings);
```

Comment:
> That whole point of moving parseDirectiveBindings out of the link phase was to avoid recreating the same objects over and over again (memory usage/gc).

`parseDirectiveBindings` is not called during link time, but during compile time. The cost of creating one new object is negligible compared to everything else that is going on at compile time.

> Can we just put an if statement around the parseDirectiveBindings call instead of changing this?

I do not understand this part, can you please elaborate?